akd/src/storage/types.rs

```diff
@@ -27,12 +27,12 @@ pub enum StorageType {
 
 /// The keys for this key-value store
 #[derive(Debug, Clone, Hash, Eq, PartialEq, Serialize, Deserialize)]
-pub struct AkdKey(pub String);
+pub struct AkdLabel(pub String);
 
-/// The types of values used in the key-value pairs of a AKD
+/// The types of value used in the key-value pairs of a AKD
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Hash)]
 #[serde(bound = "")]
-pub struct Values(pub String);
+pub struct AkdValue(pub String);
```

Comment:
I thought we were changing this to a ```Vec<u8>```. Looking at how it's utilized down-stream as well, we will want the label in binary I think as well...so

```rust
pub struct AkdLabel(pub Vec<u8>)
pub struct AkdValue(pub Vec<u8>)
```

Reply:
Yes, I'm working on that part separately after the name change, hence this comment in the PR summary above:

> This commit represents essentially just a rename of the structs, dropped in place. This is the first part of issue 104 and is pretty low stakes. I want to consider it separate from the forthcoming `String` to `Vec<u8>` conversion which is a much hairier thread to pull.

My intent here is to consider the two steps separately for ease of review, but I'm happy to combine them into a single PR if you'd prefer?

Reply:
Ah I'm sorry I missed that. 2 PRs is fine. Approving to unblock. Thanks!